I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 78, in script
    "Wow, en realidad soy muy afortunado de tener a Star a mi lado... Incluso en momentos incómodos como este en los que me pierdo en mis reflexiones y le doy tiempo de sobra para analizarme y llegar a quién sabe qué clase de juicios, 99.9'%' acertados, sobre mis emociones."
ValueError: unsupported format character ''' (0x27) at index 236

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 78, in script
    "Wow, en realidad soy muy afortunado de tener a Star a mi lado... Incluso en momentos incómodos como este en los que me pierdo en mis reflexiones y le doy tiempo de sobra para analizarme y llegar a quién sabe qué clase de juicios, 99.9'%' acertados, sobre mis emociones."
  File "D:\renpy-6.99.13-sdk\renpy\ast.py", line 624, in execute
    renpy.exports.say(who, what, interact=self.interact, *args, **kwargs)
  File "D:\renpy-6.99.13-sdk\renpy\exports.py", line 1162, in say
    what = what % tag_quoting_dict
ValueError: unsupported format character ''' (0x27) at index 236

Windows-7-6.1.7601-SP1
Ren'Py 6.99.13.2919
Project Musketer 1.0
